doc/manual.md

```diff
@@ -562,11 +727,21 @@ the same as in the standard `handshake` concepts, but this includes
 initial states for the signals. For this concept, both signals will
 be initialised to 1, or high.
 
+`never [(Transition) a, (Transition) b, ...]` 
+
+A concept used to define lists of signal transitions which cannot all have 
+occured at the same time. `(Transition)` in this 
+case will be `rise` or `fall`. These lists are used to descripe states of the
+system which do not hold for the invariant. For example, the concept
```

Comment:
I can't parse the sentence: "These lists are used to descripe states of the system which do not hold for the invariant."

(And the typo in `descripe` is not the reason.)

These lists are used to describe the invariant of the system, by specifying classes of states that must be unreachable. -- perhaps, something like this?